### PR TITLE
Implemented Mobile View

### PR DESCRIPTION
**Pull request by nimiakinleye**
#18

### CODE REVIEW

File: app-main/public/css/style.css [hunk 1]
```diff
@@ -533,11 +534,88 @@ form {
 
 @media only screen and (max-width: 1090px) {
   body {
-    min-width: 1090px;
+    /* min-width: 1090px; */
```
Comment (jrobind) [inline app-main/public/css/style.css]: Can you remove the commented out code please? (Same for rest of file)

File: app-main/public/js/index.js [hunk 2]
```diff
@@ -11,6 +11,9 @@ const avatars = [
   "octopus",
   "parrot",
 ];
+if (outerWidth < 400) {
```
Comment (jrobind) [inline app-main/public/js/index.js]: Do we still need this code? Can we remove it?

**Review by jrobind — CHANGES_REQUESTED** [review]
This is a massive improvement! Thanks @nimiakinleye – great job!

Just a few minor comments!

File: app-main/public/css/style.css (hunk 1 above)
Comment (nimiakinleye) [reply app-main/public/css/style.css]: Yh, I'll remove them.

File: app-main/public/js/index.js (hunk 2 above)
Comment (nimiakinleye) [reply app-main/public/js/index.js]: Oops, I'll remove this too.🙂

**Review by jrobind — APPROVED** [review]
Thanks for making these updates @nimiakinleye

**Comment by jrobind** [comment]
@nimiakinleye I see you've opened another PR, it's best practice to make the changes on this PR if you can rather than opening a new one 😄

**Pull request closed without merging**